packages/doenetml-worker-rust/lib-doenetml-core/src/core/general_prop/math_prop.rs

```diff
@@ -39,6 +54,11 @@ pub struct MathProp {
 #[add_dependency_data]
 #[derive(Debug, Default, PropDependencies, PropDataQueries)]
 pub struct RequiredData {
+    /// An independent state variable (that doesn't have any dependencies)
```

Comment:
`state variable` -> `prop` or `piece of data`

Reply:
This is actually a `StateProp`, the thing that used to be called "essential data". Since both become a `PropView<T>`, the distinction get muddied. It's hard to figure out what is the best way to communicate what's happening. I can try on "piece of data".